deepchecks/tabular/metric_utils/scorers.py

```diff
@@ -152,15 +137,35 @@
     'tnr_macro': make_scorer(true_negative_rate_metric, averaging_method='macro'),
     'tnr_micro': make_scorer(true_negative_rate_metric, averaging_method='micro'),
     'tnr_weighted': make_scorer(true_negative_rate_metric, averaging_method='weighted'),
+    'jaccard_macro': make_scorer(jaccard_score, average='macro', zero_division=0),
+    'jaccard_micro': make_scorer(jaccard_score, average='micro', zero_division=0),
+    'jaccard_weighted': make_scorer(jaccard_score, average='weighted', zero_division=0),
+    'jaccard_per_class': make_scorer(jaccard_score, average=None, zero_division=0),
+}
+
+binary_scorers_dict = {
+    'balanced_accuracy': get_scorer('balanced_accuracy'),
+    'average_precision': get_scorer('average_precision'),
+    'neg_brier_score': get_scorer('neg_brier_score'),
+    'precision': make_scorer(precision_score, zero_division=0),
+    'recall': make_scorer(recall_score, zero_division=0),
+    'f1': make_scorer(f1_score, zero_division=0),
+    'fpr': make_scorer(false_positive_rate_metric, averaging_method='binary'),
+    'fnr': make_scorer(false_negative_rate_metric, averaging_method='binary'),
+    'tnr': make_scorer(true_negative_rate_metric, averaging_method='binary'),
+    'jaccard': make_scorer(jaccard_score, zero_division=0),
+    'roc_auc': get_scorer('roc_auc'),
+    'neg_log_loss': make_scorer(log_loss, greater_is_better=False, needs_proba=True, labels=[0, 1]),
+    **common_classification_metrics
+}
+
+multiclass_scorers_dict = {
     'roc_auc_per_class': make_scorer(roc_auc_per_class, needs_proba=True),
```

Comment:
maybe move all per class methods here?

Reply:
will look better for monitoring

Reply:
But it think the user should be able to have per class also for the binary case

Reply:
I think is a rarer case.
For the case you described - It is most likely representable by other metrics for the positive class
For the case I mentioned - I can be confusing

Anyhow I do not have a strong preference

Reply:
mm good point, will move them to multiclass only